### PR TITLE
Remove dependency to 'ocean.core.Test'

### PR DESCRIPTION
**Pull request by Geod24**
```
Now that '-checkaction=context' is usable (and the default),
we do not need to use this function anymore and can rely on regular assert.
```

### CODE REVIEW

**Comment by Geod24** [comment]
Rebased

File: source/agora/consensus/EnrollmentManager.d [hunk 1]
```diff
@@ -1073,26 +1072,27 @@ unittest
...
-    test!"=="(man.validator_set.countActive(EnrollAt9 + 1), 0);
-    test!"=="(man.validator_set.getEnrolledHeight(EnrollAt9, utxo_hash), ulong.max);
+    assert(man.validator_set.countActive(EnrollAt9 + 1) == 0);
+    assert(man.validator_set.getEnrolledHeight(EnrollAt9, utxo_hash) == ulong.max);
     // Add removed enrollment to the pool with the new height
     assert(man.addEnrollment(
         EnrollmentManager.makeEnrollment(utxo_hashes[1], pairs[1], EnrollAt9, params.ValidatorCycle),
             pairs[1].address, EnrollAt9, &utxo_set.peekUTXO));
     // The expired enrollments in the pool from height 1 are cleared on this next call
     // to get enrollments at a highter height. We do have an enrollment at height 9
```
Comment (kchulj) [inline source/agora/consensus/EnrollmentManager.d:1082]:
```suggestion
    // to get enrollments at a higher height. We do have an enrollment at height 9
```

File: source/agora/consensus/EnrollmentManager.d [hunk 2]
```diff
@@ -1073,26 +1072,27 @@ unittest
...
     // One Enrollment was moved to validator set
-    test!"=="(man.validator_set.countActive(EnrollAt9 + 1), 1);
+    assert(man.validator_set.countActive(EnrollAt9 + 1) == 1);
     // Check last block of cycle is still active
-    test!"=="(man.validator_set.countActive(EnrollAt9 + params.ValidatorCycle), 1);
+    assert(man.validator_set.countActive(EnrollAt9 + params.ValidatorCycle) == 1);
     // Check block in next cycle is no longer active
-    test!"=="(man.validator_set.countActive(EnrollAt9 + params.ValidatorCycle + 1), 0);
+    assert(man.validator_set.countActive(EnrollAt9 + params.ValidatorCycle + 1) == 0);
     // Pool shoud now be empty
```
Comment (kchulj) [inline source/agora/consensus/EnrollmentManager.d:1094]:
```suggestion
    // Pool should now be empty
```

**Review by kchulj — APPROVED** [review]
Took a while to review..
LGTM + nits

**Comment by Geod24** [comment]
@kchulj : As our guideline is to have atomic commits, and those lines are unrelated, I'm not going to change them in this commit. Sometimes it's okay to change not-directly-related code in the same commit, for example if you are going to change the line anyway: e.g. if while changing `test!"..."` => `assert` we spotted a typo in a message, it would be okay to have the following diff:
```diff
- test!"=="(a, b, "Expessions are not equal");
+ assert(a == b", Expressions are not equal");
```

Because, you know, we're touching the line anyway.

However, that doesn't mean we shouldn't fix those: https://github.com/bosagora/agora/pull/2159

**Pull request merged**